addon/enums/enumerable.js

```diff
@@ -0,0 +1,25 @@
+import EmberObject from '@ember/object';
+
+export default EmberObject.extend({
```

Comment:
should we add tests for these 2 functions?

Reply:
I don't feel we need to, but I can add :)

Reply:
I'll add it in a further PR to do not delay the release of 0.4.0